### PR TITLE
avatar badge component

### PR DESCRIPTION
**Pull request by som749898**
avatar badge component completed
live preview: https://awesome-lalande-fd3882.netlify.app/
NOT RESPONSIVE

### CODE REVIEW

**Review by maitrakhatri — APPROVED** [review]
looks good to me

File: CSS/color.css [hunk 1]
```diff
@@ -17,27 +16,27 @@
...
+  background-color: var(--success-color);
+  color: var(--secondary-color);
 }
 
 .box:nth-child(6) {
-  background-color: #ffe7c5;
-  color: #666;
+  background-color: var(--warning-color);
+  color: var(--secondary-color);
 }
```
Comment (rohanmathur91) [inline CSS/color.css]: Use classes for different conditions, nth-child might cause a problem in styling in the future.

File: CSS/main.css [hunk 2]
```diff
@@ -31,6 +31,12 @@ body {
   margin: 0px auto;
 }
 
+#navbar {
+  padding: 20px 10px;
```
Comment (rohanmathur91) [inline CSS/main.css]: use rem if possible

File: CSS/main.css [hunk 3]
```diff
@@ -31,6 +31,12 @@ body {
...
+  display: flex;
+  justify-content: center;
+}
+
 #icon {
   font-size: 32px;
```
Comment (rohanmathur91) [inline CSS/main.css]: same as above.

File: CSS/main.css [hunk 4]
```diff
@@ -39,12 +45,6 @@ body {
   color: #fff;
 }
```
Comment (rohanmathur91) [inline CSS/main.css:67]: Have a variable for color

File: CSS/main.css [hunk 5]
```diff
@@ -61,16 +61,24 @@ body {
...
 
+.menu-list a:hover {
+  color: #276EF1;
```
Comment (rohanmathur91) [inline CSS/main.css]: same as above

**Review by rohanmathur91 — COMMENTED** [review]
Use variable for color, rest look good.

File: CSS/color.css (hunk 1 above)
Comment (som749898) [reply CSS/color.css]: I have provided class name to boxes instead of using pseudo-class

File: CSS/main.css (hunk 2 above)
Comment (som749898) [reply CSS/main.css]: converted pixel to rem

File: CSS/main.css (hunk 3 above)
Comment (som749898) [reply CSS/main.css]: converted pixel to rem

File: CSS/main.css (hunk 5 above)
Comment (som749898) [reply CSS/main.css]: added css variable

File: CSS/color.css (hunk 1 above)
Comment (rohanmathur91) [reply CSS/color.css]: Than its fine.

**Review by rohanmathur91 — APPROVED** [review]
Looks goos to me.

**Pull request merged**